src/stdafx.h

```diff
@@ -430,19 +418,34 @@ assert_compile(SIZE_MAX >= UINT32_MAX);
 #	define CloseConnection OTTD_CloseConnection
 #endif /* __APPLE__ */
 
+#if defined(__GNUC__) || defined(__clang__)
+#define likely(x)       __builtin_expect(!!(x), 1)
```

Comment:
I think this is missing spaces between `#` and `defined` .. or tabs .. or what silly was it we do :)